elastic/tests/test_elastic.py

```diff
@@ -191,6 +192,298 @@ def test_pshard_metrics(dd_environment, aggregator):
+        "elasticsearch.thread_pool.get.active",
+        "elasticsearch.thread_pool.get.completed",
+        "elasticsearch.thread_pool.get.completed.count",
+        "elasticsearch.thread_pool.get.queue",
+        "elasticsearch.thread_pool.get.rejected",
+        "elasticsearch.thread_pool.get.rejected.count",
+        "elasticsearch.thread_pool.get.threads",
+        "elasticsearch.thread_pool.get.threads.count",
+        "elasticsearch.thread_pool.listener.active",
+        "elasticsearch.thread_pool.listener.queue",
+        "elasticsearch.thread_pool.listener.rejected",
+        "elasticsearch.thread_pool.listener.rejected.count",
+        "elasticsearch.thread_pool.listener.threads",
+        "elasticsearch.thread_pool.listener.threads.count",
+        "elasticsearch.thread_pool.management.active",
+        "elasticsearch.thread_pool.management.completed",
+        "elasticsearch.thread_pool.management.completed.count",
+        "elasticsearch.thread_pool.management.queue",
+        "elasticsearch.thread_pool.management.rejected",
+        "elasticsearch.thread_pool.management.rejected.count",
+        "elasticsearch.thread_pool.management.threads",
+        "elasticsearch.thread_pool.management.threads.count",
+        "elasticsearch.thread_pool.refresh.active",
+        "elasticsearch.thread_pool.refresh.completed",
+        "elasticsearch.thread_pool.refresh.completed.count",
+        "elasticsearch.thread_pool.refresh.queue",
+        "elasticsearch.thread_pool.refresh.rejected",
+        "elasticsearch.thread_pool.refresh.rejected.count",
+        "elasticsearch.thread_pool.refresh.threads",
+        "elasticsearch.thread_pool.refresh.threads.count",
+        "elasticsearch.thread_pool.search.active",
+        "elasticsearch.thread_pool.search.completed",
+        "elasticsearch.thread_pool.search.completed.count",
+        "elasticsearch.thread_pool.search.queue",
+        "elasticsearch.thread_pool.search.rejected",
+        "elasticsearch.thread_pool.search.rejected.count",
+        "elasticsearch.thread_pool.search.threads",
+        "elasticsearch.thread_pool.search.threads.count",
+        "elasticsearch.thread_pool.snapshot.active",
+        "elasticsearch.thread_pool.snapshot.completed",
+        "elasticsearch.thread_pool.snapshot.completed.count",
+        "elasticsearch.thread_pool.snapshot.queue",
+        "elasticsearch.thread_pool.snapshot.rejected",
+        "elasticsearch.thread_pool.snapshot.rejected.count",
+        "elasticsearch.thread_pool.snapshot.threads",
+        "elasticsearch.thread_pool.snapshot.threads.count",
+        "elasticsearch.thread_pool.warmer.active",
+        "elasticsearch.thread_pool.warmer.completed",
+        "elasticsearch.thread_pool.warmer.queue",
+        "elasticsearch.thread_pool.warmer.rejected",
+        "elasticsearch.thread_pool.warmer.threads",
+    }
+    for m_name in all_other_metrics:
+        aggregator.assert_metric(m_name)
+    aggregator.assert_all_metrics_covered()
```

Comment:
```suggestion
```
Sorry for the noise here, it's something we should add more broadly to this integration, that is out of scope of this PR